ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -108,9 +108,9 @@ export const getFreeCreditUsage = (profile: Profile): string => {
 }
 
 // returns the updated profile value only if it has changed
-export const getUpdatedProfileValue = (oldProfile: Profile, updatedProfile: Profile, attribute: string) => {
-  const oldValue = fp.get([attribute], oldProfile);
-  const updatedValue = fp.get([attribute], updatedProfile);
+export const getUpdatedProfileValue = (oldProfile: Profile, updatedProfile: Profile, attributePath: string[]) => {
```

Comment:
this change is not strictly needed.  it relates to a different change which was reverted.  Good to keep though b/c more flexible